api/tests/unit/app/test_unit_app_utils.py

```diff
@@ -76,6 +77,90 @@ def path_side_effect(file_path: str) -> mocker.MagicMock:
     assert result == {
         "ci_commit_sha": "unknown",
         "image_tag": "unknown",
+        "has_email_provider": False,
         "is_enterprise": True,
         "is_saas": False,
     }
+
+
+def test_get_version_info_with_email_config_smtp(mocker: MockerFixture) -> None:
+
+    mocker.patch(
+        "app.utils.EMAIL_BACKEND", "django.core.mail.backends.smtp.EmailBackend"
+    )
+    mocker.patch("app.utils.EMAIL_HOST_USER", "user")
```

Comment:
Instead of using `mocker.patch()` I'd point you towards using the fixture `settings: SettingsWrapper` which will override these constants all at once and saving the test suite from getting into an inconsistent state where they are sometimes set and other times unset.

Of course this suggestion applies to all of the tests.